tests/unit/test_connection_manager.py

```diff
@@ -24,3 +24,9 @@ def test_get_response(self, state, result):
         assert isinstance(response, AdapterResponse)
         assert response.code == result
         assert response.rows_affected == 1
+
+    def test_data_type_code_to_name(self):
+        cm = AthenaConnectionManager(mock.MagicMock())
+        assert cm.data_type_code_to_name("array<string>") == "ARRAY"
+        assert cm.data_type_code_to_name("map<int, boolean>") == "MAP"
+        assert cm.data_type_code_to_name("DECIMAL(3, 7)") == "DECIMAL"
```

Comment:
do we have cases where varchar(45) is returned? in such case what it needs to be returned? based on function implementation seems `VARCHAR`